libs/mrwebrtc/src/media/audio_track_read_buffer.h

```diff
@@ -56,12 +56,14 @@ class AudioTrackReadBuffer {
   };
   // Incoming frames received from webrtc - see also buffer_
   std::deque<Frame> frames_;
-  // protects frames_ in Read() and audioFrameCallback()
+  // protects frames_ and has_overrun_ in Read() and audioFrameCallback()
   std::mutex frames_mutex_;
   // max ms of audio data stored in frames_
-  int buffer_ms_ = 0;
+  int buffer_ms_{};
```

Comment:
nit: `buffering_ms_` or `buffer_size_ms_`? Sounds like this could be a pointer to a buffer, whereas it's a buffering length.

Reply:
Fixed